pandas-stubs/core/indexing.pyi

```diff
@@ -69,6 +73,14 @@ class _LocIndexerFrame:
     @overload
     def __setitem__(self, idx: _StrLike, value: Union[float, Series, _np.ndarray],) -> None: ...
     @overload
+    def __setitem__(
+        self, idx: Tuple[_np.int64, str], value: Union[_np.ndarray, List[int], float]
```

Comment:
if you're indexing with an `int` (or rather `_np.int64`), then you should probably be using `.iloc`.

(And why only allow `List[int]` as value? what about `List[float]`?)

Reply:
{ValueError}Location based indexing can only have [integer, integer slice (START point is INCLUDED, END point is EXCLUDED), listlike of integers, boolean array] types